gremlin-javascript/src/main/javascript/gremlin-javascript/test/integration/client-behavior-tests.js

```diff
@@ -21,58 +21,56 @@
 
 const assert = require('assert');
 const helper = require('../helper');
-const {getUserAgent} = require("../../lib/utils");
+const { getUserAgent } = require('../../lib/utils');
 
 let client;
 let settings;
 
 describe('Client', function () {
-    before(function () {
-        client = helper.getGremlinSocketServerClient('gmodern');
-        settings = helper.getGremlinSocketServerSettings();
-        return client.open();
-    });
-    after(function () {
-        return client.close();
-    });
-    describe('#submit()', function () {
-        it('should reconnect after server closes connection', async function () {
-            let connectionClosed = false;
-            await client.submit('1', null, {requestId: settings.CLOSE_CONNECTION_REQUEST_ID})
-                .catch(function(error){
-                    assert.equal(error.toString(), 'Error: Connection has been closed.');
-                    connectionClosed = true;
-                });
+  before(function () {
```

Comment:
I noticed that for the test files a lot of the indentation was changed. I'm assuming this was done by a linter but do you mind reverting it back to what it was before. It's a bit difficult to read the Gremlin queries when they are split over multiple lines.

Reply:
Yes, this is from prettier formatting. I've got my editor setup to automatically format on save if the project have prettier setted up, which is project have. I think it's a good idea to enforce formatting rule, even on CI.

But if an exception should be made for the test files, should I add all test files to `.prettierignore` then?

Reply:
But in saying that, I've reverted all the style changes here for now.